trecs/components/users.py

```diff
@@ -170,6 +175,7 @@ def __init__(
         verbose=False,
         seed=None,
         attention_exp=0.0,
+        repeat_items=True,
```

Comment:
Can we find another term for `repeat_items`? My concern here is that it seems to imply items are repeated from the recommender side, but that is not what this variable controls. How about something like `repeat_interactions`?

Reply:
Good idea! I like that modification. Thanks for the suggestion!